### PR TITLE
[SSAT] Add ssat to test nnstreamer

### PR DESCRIPTION
**Pull request by jijoongmoon**
# PR Description
Currently NNstreamer uses ssat to run the test cases. It is in the
recipes-devtools dir.

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: recipes-devtools/ssat/ssat_1.0.0.bb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+SUMMARY = "Shell Script Automated Tester (uint testing executable files)"
+DESCRIPTION = "Shell Script Automated Tester (uint testing executable files)"
```
Comment (myungjoo) [inline recipes-devtools/ssat/ssat_1.0.0.bb]: ```s/uint/unit/```

File: recipes-devtools/ssat/ssat_1.0.0.bb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+SECTION = "AI"
+LICENSE = "LGPLv2+"
```
Comment (myungjoo) [inline recipes-devtools/ssat/ssat_1.0.0.bb]: It's APACHE 2.0

File: recipes-devtools/ssat/ssat_1.0.0.bb [hunk 3]
```diff
@@ -0,0 +1,26 @@
+SUMMARY = "Shell Script Automated Tester (uint testing executable files)"
+DESCRIPTION = "Shell Script Automated Tester (uint testing executable files)"
+
+SECTION = "AI"
```
Comment (myungjoo) [inline recipes-devtools/ssat/ssat_1.0.0.bb]: It's rather "Development" or "Test Suite"

File: recipes-devtools/ssat/ssat_1.0.0.bb (hunk 1 above)
Comment (jijoongmoon) [reply recipes-devtools/ssat/ssat_1.0.0.bb]: fixed.

File: recipes-devtools/ssat/ssat_1.0.0.bb (hunk 2 above)
Comment (jijoongmoon) [reply recipes-devtools/ssat/ssat_1.0.0.bb]: fixed.

File: recipes-devtools/ssat/ssat_1.0.0.bb (hunk 3 above)
Comment (jijoongmoon) [reply recipes-devtools/ssat/ssat_1.0.0.bb]: fixed.

**Pull request merged**